ui/raidboss/data/04-sb/raid/o8s.js

```diff
@@ -188,7 +188,7 @@ export default {
           en: 'Future: Stack and Through',
           de: 'Zukunft: Sammeln und Durchlaufen',
           fr: 'Futur : Stack et traversez',
-          ja: '未来: シェア後ボス通り抜ける',
+          ja: '未来: シェア後ボスを通り抜ける',
```

Comment:
```suggestion
          ja: '未来: 頭割り後ボスを通り抜ける',
```
I think to describe "damage share" gimmick, we don't use "シェア" anymore, all use "頭割り" instead.